linera-views/Cargo.toml

```diff
@@ -38,12 +38,12 @@ sha3 = { workspace = true }
 thiserror = { workspace = true }
 linked-hash-map = { workspace = true }
 tempfile = { workspace = true }
+once_cell = { workspace = true }
```

Comment:
I think that crate is obsolete now: https://doc.rust-lang.org/stable/std/cell/struct.OnceCell.html